src/main/java/com/epam/lab/paymentsystem/entities/User.java

```diff
@@ -24,7 +24,7 @@
   @Column(name = "login")
   private String login;
 
-  @ManyToOne(cascade = CascadeType.ALL)
+  @ManyToOne(cascade = CascadeType.REFRESH)
```

Comment:
Why to change?

Reply:
We decided that you can use this type, the main thing that would be the same everywhere